base/bool.jl

```diff
@@ -77,6 +77,7 @@ sign(x::Bool) = x
 abs(x::Bool) = x
 abs2(x::Bool) = x
 iszero(x::Bool) = !x
+isone(x::Bool) = x
```

Comment:
This will be handled by the fallback method
```julia
julia> true == one(true)
true

julia> false == one(false)
false
```
but perhaps keep here for completeness? Others can decide...